src/react/components/pages/editorPage/canvas.tsx

```diff
@@ -938,14 +946,19 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
         });
     }
 
-    private loadOcr = async () => {
+    private rerunOcr = () => {
```

Comment:
I don't this this wrapper function is necessary, you could just pass loadOcr as prop to command bar right?